app.js

```diff
@@ -57,6 +57,10 @@ app.use(function(err, req, res, next) {
   res.send(err);
 });
 
+app.get('/stay-secure/*', function (request, response) {
+  response.sendFile(path.join(__dirname, '/public/secure-index.html'));
```

Comment:
This is a quick hack because this new page needs a separate set of og tags and title. Right now this app doesn't serve up a configurable index page.